app/views/RoomView/styles.js

```diff
@@ -66,5 +69,13 @@ export default StyleSheet.create({
 	previewMode: {
 		fontSize: 16,
 		...sharedStyles.textMedium
+	},
+	icon: {
+		width: 22,
+		height: 22
+	},
+	textContainer: {
```

Comment:
Try to have the same margin on `Left` and `Right` side.
![paint](https://user-images.githubusercontent.com/29778115/80386629-033fef80-887e-11ea-8f6c-48a29f834822.png)